src/config/mnemonic.ts

```diff
@@ -46,7 +47,9 @@ export const configExtender: ConfigExtender = (finalConfig, userConfig) => {
       const { rpc, keys } = byNetworkMnemonic
       if (!finalNetwork.url && rpc) finalNetwork.url = rpc
       if (!finalNetwork.accounts && keys)
-        finalNetwork.accounts = ensureHexEncoding(keys)
+        finalNetwork.accounts = isArray(keys)
```

Comment:
Ah, rather than importing this from lodash, we can just use `Array.isArray()`.

Reply:
Good point!